plugins/copybuffer/README.md

```diff
@@ -0,0 +1,14 @@
+# `copybuffer` plugin
+
+This plugin adds a keybinding that copies the command line buffer contents to
+the system clipboard.
+
+```zsh
+plugins=(... copybuffer)
+```
+
+## Features
+
+| Keybinding | Command        | Description                                        |
+|------------|----------------|----------------------------------------------------|
+| `ctrl-o`   | `copybuffer`   | Copies the command line buffer to system clipboard |
```

Comment:
"command line buffer" is too close to implementation, I'd prefer "currently written command line (`$BUFFER`)", which has both the abstract and the low-level information.